packages/charts/src/charts/donut/donut-label.component.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, memo } from 'react';
+
+import { ColorAdjuster, Typography } from '@keen.io/ui-core';
+
+type Props = {
```

Comment:
I believe this component could be easily extracted - to be re-used in Pie and Donut charts.